archivist/confirmer.py

```diff
@@ -94,20 +95,18 @@ def _wait_for_confirmation(self: Managers, identity: str) -> ReturnTypes:
             f"cannot confirm {identity} as confirmation_status is not present"
         )
 
-    if entity[CONFIRMATION_STATUS] == ConfirmationStatus.FAILED.name:
+    status = entity[CONFIRMATION_STATUS]
+    if status == ConfirmationStatus.FAILED.name:
         raise ArchivistUnconfirmedError(
             f"confirmation for {identity} FAILED - this is unusable"
         )
 
-    # Simple hash
-    if entity[CONFIRMATION_STATUS] == ConfirmationStatus.CONFIRMED.name:
+    # Simple hash and merkleLog
+    if status == ConfirmationStatus.CONFIRMED.name:
         return entity
 
     # merkle_log
-    if (
-        ConfirmationStatus[entity[CONFIRMATION_STATUS]].value
-        >= ConfirmationStatus.COMMITTED.value
-    ):
+    if ConfirmationStatus[status].value >= ConfirmationStatus.COMMITTED.value:
```

Comment:
I am a bit weary of >= when we already know we don't adhere to an increased trust level the higher the enum.

Why not == Committed?

We have a ticket for thinking about introducing the concept of 'trust level greater than'.

Reply:
weary?

this also covers the case of UNEQUIVOCAL which I know we don't have just yet. I that is unimportant then I will change...

Reply:
fixed it